### PR TITLE
Fix warning in php7: count on null in options importer

### PR DESCRIPTION
**Pull request by marvinhinz**
Fixes: Warning: count(): Parameter must be an array or an object that implements Countable in vendor/macopedia/magmi2/magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php on line 124

### CODE REVIEW

File: magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php [hunk 1]
```diff
@@ -121,7 +121,7 @@ public function createOptionValues($field, $sids, $valarr)
         $cvalarr = count($valarr);
         for ($i = 0; $i < $cvalarr; $i++) {
             $val = $valarr[$i];
-            if ($i < count($optionTypeIds)) {
+            if ($optionTypeIds && $i < count($optionTypeIds)) {
```
Comment (tmotyl) [inline magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php]: what about changing the default value to empty array in https://github.com/macopedia/magmi-m2/blob/58a7d9769edf5759eb00f1f358c5dbed2085ef9c/magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php#L39 ?
like:
```
        return isset($this->_opttypeids[$field]) ? $this->_opttypeids[$field] : [];

```

Comment (marvinhinz) [reply magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php]: Actually i did that first too, but i thought maybe an empty array could be unexpected somewhere else. I didn't look much into it, i'm okay with both solutions :) What do you prefer?

Comment (tmotyl) [reply magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php]: I've checked and this method is used in the single place, so its safe to change it

Comment (marvinhinz) [reply magmi/plugins/extra/itemprocessors/customoptions/pablo_customoptions.php]: Okay, i changed the PR

**Pull request merged**